synapse/handlers/register.py

```diff
@@ -123,10 +126,12 @@ def check_username(self, localpart, guest_access_token=None,
 
         self.check_user_id_not_appservice_exclusive(user_id)
 
-        if len(user_id) > 256:
+        if len(user_id) > MAX_ALIAS_LENGTH:
```

Comment:
this should be MAX_USERID_LENGTH

Reply:
For reference, this looks fixed on `master` now [`synapse/handlers/register.py#L104`](https://github.com/matrix-org/synapse/blob/31d721fbf6655080235003b5576110d477fa2353/synapse/handlers/register.py#L104)